src/AppBundle/Factory/ClubFactory.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
This Factory is never used so -> YAGNI. Please remove it

Reply:
You are right. Thanks for code review.

Reply:
That's another problem. Has full address with `Club` is weird. To be better to create another special entity.

Reply:
there is a Place Entity

Reply:
ok, I didn't see.